src/dropin.js

```diff
@@ -169,6 +249,12 @@ Dropin.prototype._disableErroredPaymentMethods = function () {
   }.bind(this));
 };
 
+/**
+ * Requests a payment method object, that includes the payment method nonce used by by the [Braintree Server SDKs](https://developers.braintreepayments.com/start/hello-server/). The structure of this object varies by type: a {@link Dropin~cardPaymentMethodPayload|cardPaymentMethodPayload} is returned when the payment method is a card, a {@link Dropin~paypalPaymentMethodPayload|paypalPaymentMethodPayload} is returned when the payment method is a PayPal account. If a payment method is not available, an error will appear in the UI and and error will be returned in the callback.
+ * @public
+ * @param {callback} callback The first argument will be an error if no payment method is available and will otherwise be null. The second argument will be an object containing a payment method nonce; either a {@link Dropin~paypalPaymentMethodPayload|paypalPaymentMethodPayload} or a {@link Dropin~paypalPaymentMethodPayload|paypalPaymentMethodPayload}.
```

Comment:
I misread this at first as the payment method nonce being a paypal or card payload. Not sure if this needs to be adjusted or not.